build.gradle

```diff
@@ -191,9 +191,9 @@ dependencies {
     compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-log4j2', version: '2.4.0'
 
     testCompile 'junit:junit:4.12'
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.5.1'
-    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.5.1'
-    testCompile 'org.junit.platform:junit-platform-launcher:1.5.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.4.2'
```

Comment:
From the 5.5 release notes:
https://junit.org/junit5/docs/current/release-notes/index.html#release-notes-5.5.0
>Explicit Java module descriptors

Probably we need to import the modules

Reply:
https://github.com/junit-team/junit5/blob/master/README.md#junit-platform

Reply:
Yap, should work now.